shared/tsconfig.json

```diff
@@ -3,11 +3,13 @@
     "noUncheckedIndexedAccess": true,
     "declaration": true,
     "esModuleInterop": true,
-    "target": "ES2020",
+    "strict": true,
+    "target": "es2019",
+    "lib": ["ES2019"],
+    "module": "commonjs",
```

Comment:
or this?!

Reply:
actually i think we need this as well 😂  without this it build files using ESM. node v14.13.0 and above has support for ESM but we need to modify package.json type to be module and do additional stuff :sadsad:

big ups to this guy: https://stackoverflow.com/questions/38296667/getting-unexpected-token-export